### PR TITLE
ebuild: add page

### LINKED ISSUE #646: page request: ebuild
What about small ebuild for gentoo?

### PR DESCRIPTION
**Pull request by pxgamer**
Closes #646

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/ebuild.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- Extract the sources to a temporary build directory:
+
+`ebuild {{path/to/file.ebuild}} unpack`
+
+- Compile the extracted sources:
+
+`ebuild {{path/to/file.ebuild}} compile`
+
+- Install the package to a temporary install directory:
```
Comment (sbrl) [inline pages/linux/ebuild.md]: I guess this tool isn't for installing things for real?

File: pages/linux/ebuild.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+`ebuild {{path/to/file.ebuild}} install`
+
+- Install the temporary files to the live filesystem:
+
+`ebuild {{path/to/file.ebuild}} qmerge`
+
+- Fetch, unpack, compile, install and qmerge the specified ebuild file:
```
Comment (agnivade) [inline pages/linux/ebuild.md]: You mention qmerge here, but the example has merge. Is this right ?

Comment (pxgamer) [reply pages/linux/ebuild.md]: Yes. Qmerge is just the step for merging installs to the live filesystem which `merge` includes.

File: pages/linux/ebuild.md (hunk 1 above)
Comment (pxgamer) [reply pages/linux/ebuild.md]: If you run `qmerge` then it does install the package to the live filesystem. The actual `install` command just installs them in a sandboxed container.

Comment (sbrl) [reply pages/linux/ebuild.md]: Doesn't `live filesystem`  refer to an in-memory filesystem that's wiped when the machine shuts down though?

Comment (pxgamer) [reply pages/linux/ebuild.md]: Sorry for the late reply, I didn't actually know that. :thinking:

Comment (sbrl) [reply pages/linux/ebuild.md]: Neither do I! I'm just going by the terminology I'm familiar with - i.e. a "live CD" boots into a "live session", which has a "live file system" - and everything gets wiped on shutdown 'cause it's all stored in RAM.

**Pull request merged**